src/features/auth/components/ForgotPassword/PasswordForm.js

```diff
@@ -6,7 +6,7 @@ import { useHistory } from 'react-router-dom';
 import { object, ref, string } from 'yup';
 import PropTypes from 'prop-types';
 
-import { useAuth } from 'features/auth';
+import { useAuth } from '@loopstudio/react-auth';
```

Comment:
This import should be above with the libraries' ones, right?

Reply:
you are right, I'll fix that, thanks!